unstructured/ingest/pipeline/pipeline.py

```diff
@@ -86,6 +86,17 @@ def run(self):
         copier(iterable=partitioned_jsons)
 
         if self.write_node:
+            logger.info(
+                f"uploading elements from {len(partitioned_jsons)} "
+                "document(s) to the destination"
+            )
+
+            write_config = self.write_node.dest_doc_connector.write_config
+            if hasattr(write_config, "num_processes"):
```

Comment:
Can we add this log inside the pinecone destination connector so we don't have to use `hasattr` to dynamically check if that exists?

Reply:
I think this log should ideally reside in pipeline, 

however since we had disagreement on `num_processes` and since the option is not yet available for other connectors, addressed with: [move log for number of (upload) processes from pipeline to connector](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/1cf12903494929b4ae59c6a7210461dbc7898685)